4-projects/modules/infra_pipelines/cloudbuild_builder/Dockerfile

```diff
@@ -32,8 +32,8 @@ RUN apt-get update && \
     sha256sum -c terraform_SHA256SUMS --status && \
     unzip terraform_linux_amd64.zip -d /builder/terraform && \
     rm -f terraform_linux_amd64.zip && \
-    gsutil cp gs://terraform-validator/releases/${ENV_TERRAFORM_VALIDATOR_RELEASE}/terraform-validator-linux-amd64 /builder/terraform/terraform-validator && \
-    chmod +x /builder/terraform/terraform-validator && \
+    /builder/google-cloud-sdk/bin/gcloud components update && \
+    /builder/google-cloud-sdk/bin/gcloud components install terraform-tools && \
```

Comment:
Hi @iyabchen Thanks for the PR.

This change to add
```
  /builder/google-cloud-sdk/bin/gcloud components install terraform-tools && \
```

would also need to be done at the bootstrap module before been able to switch to `gcloud beta terraform vet`  

since the image 

https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/v5.1.0/modules/cloudbuild/cloudbuild_builder/Dockerfile

is used to run the cloud builds with the `tf-wrapper.sh` script for the steps 1 to 4

Reply:
https://github.com/terraform-google-modules/terraform-google-bootstrap/pull/156

Reply:
test failure might be related to https://github.com/terraform-google-modules/terraform-google-bootstrap/pull/148.